### ISSUE TITLE
Backend: Add Session Table to SQL, add Session ID to Review table

### ISSUE THREAD

**Issue by DarkPurple141** [SmartCourse/app#284]
As above:

Also, ensure backend routes accomodate the session being created on `POST` request.

**Comment by DarkPurple141** [comment]
Resolved by #317

**Comment by DarkPurple141** [comment]
```json
[{"id":1,"shortName":"05X","longName":"Summer 2005","year":2005},{"id":2,"shortName":"05S1","longName":"Semester 1 2005","year":2005},{"id":3,"shortName":"05S2","longName":"Semester 2 2005","year":2005},{"id":4,"shortName":"06X","longName":"Summer 2006","year":2006},{"id":5,"shortName":"06S1","longName":"Semester 1 2006","year":2006},{"id":6,"shortName":"06S2","longName":"Semester 2 2006","year":2006},{"id":7,"shortName":"07X","longName":"Summer 2007","year":2007},{"id":8,"shortName":"07S1","longName":"Semester 1 2007","year":2007},{"id":9,"shortName":"07S2","longName":"Semester 2 2007","year":2007},{"id":10,"shortName":"08X","longName":"Summer 2008","year":2008},{"id":11,"shortName":"08S1","longName":"Semester 1 2008","year":2008},{"id":12,"shortName":"08S2","longName":"Semester 2 2008","year":2008},{"id":13,"shortName":"09X","longName":"Summer 2009","year":2009},{"id":14,"shortName":"09S1","longName":"Semester 1 2009","year":2009},{"id":15,"shortName":"09S2","longName":"Semester 2 2009","year":2009},{"id":16,"shortName":"10X","longName":"Summer 2010","year":2010},{"id":17,"shortName":"10S1","longName":"Semester 1 2010","year":2010},{"id":18,"shortName":"10S2","longName":"Semester 2 2010","year":2010},{"id":19,"shortName":"11X","longName":"Summer 2011","year":2011},{"id":20,"shortName":"11S1","longName":"Semester 1 2011","year":2011},{"id":21,"shortName":"11S2","longName":"Semester 2 2011","year":2011},{"id":22,"shortName":"12X","longName":"Summer 2012","year":2012},{"id":23,"shortName":"12S1","longName":"Semester 1 2012","year":2012},{"id":24,"shortName":"12S2","longName":"Semester 2 2012","year":2012},{"id":25,"shortName":"13X","longName":"Summer 2013","year":2013},{"id":26,"shortName":"13S1","longName":"Semester 1 2013","year":2013},{"id":27,"shortName":"13S2","longName":"Semester 2 2013","year":2013},{"id":28,"shortName":"14X","longName":"Summer 2014","year":2014},{"id":29,"shortName":"14S1","longName":"Semester 1 2014","year":2014},{"id":30,"shortName":"14S2","longName":"Semester 2 2014","year":2014},{"id":31,"shortName":"15X","longName":"Summer 2015","year":2015},{"id":32,"shortName":"15S1","longName":"Semester 1 2015","year":2015},{"id":33,"shortName":"15S2","longName":"Semester 2 2015","year":2015},{"id":34,"shortName":"16X","longName":"Summer 2016","year":2016},{"id":35,"shortName":"16S1","longName":"Semester 1 2016","year":2016},{"id":36,"shortName":"16S2","longName":"Semester 2 2016","year":2016},{"id":37,"shortName":"17X","longName":"Summer 2017","year":2017},{"id":38,"shortName":"17S1","longName":"Semester 1 2017","year":2017},{"id":39,"shortName":"17S2","longName":"Semester 2 2017","year":2017},{"id":40,"shortName":"18X","longName":"Summer 2018","year":2018},{"id":41,"shortName":"18S1","longName":"Semester 1 2018","year":2018},{"id":42,"shortName":"18S2","longName":"Semester 2 2018","year":2018},{"id":43,"shortName":"19X","longName":"Summer 2019","year":2019},{"id":44,"shortName":"19T1","longName":"Term 1 2019","year":2019},{"id":45,"shortName":"19T2","longName":"Term 2 2019","year":2019},{"id":46,"shortName":"19T3","longName":"Term 3 2019","year":2019},{"id":47,"shortName":"20X","longName":"Summer 2020","year":2020},{"id":48,"shortName":"20T1","longName":"Term 1 2020","year":2020},{"id":49,"shortName":"20T2","longName":"Term 2 2020","year":2020},{"id":50,"shortName":"20T3","longName":"Term 3 2020","year":2020},{"id":51,"shortName":"21X","longName":"Summer 2021","year":2021},{"id":52,"shortName":"21T1","longName":"Term 1 2021","year":2021},{"id":53,"shortName":"21T2","longName":"Term 2 2021","year":2021},{"id":54,"shortName":"21T3","longName":"Term 3 2021","year":2021},{"id":55,"shortName":"22X","longName":"Summer 2022","year":2022},{"id":56,"shortName":"22T1","longName":"Term 1 2022","year":2022},{"id":57,"shortName":"22T2","longName":"Term 2 2022","year":2022},{"id":58,"shortName":"22T3","longName":"Term 3 2022","year":2022},{"id":59,"shortName":"23X","longName":"Summer 2023","year":2023},{"id":60,"shortName":"23T1","longName":"Term 1 2023","year":2023},{"id":61,"shortName":"23T2","longName":"Term 2 2023","year":2023},{"id":62,"shortName":"23T3","longName":"Term 3 2023","year":2023},{"id":63,"shortName":"24X","longName":"Summer 2024","year":2024},{"id":64,"shortName":"24T1","longName":"Term 1 2024","year":2024},{"id":65,"shortName":"24T2","longName":"Term 2 2024","year":2024},{"id":66,"shortName":"24T3","longName":"Term 3 2024","year":2024},{"id":67,"shortName":"05X","longName":"Summer 2005","year":2005},{"id":68,"shortName":"05S1","longName":"Semester 1 2005","year":2005},{"id":69,"shortName":"05S2","longName":"Semester 2 2005","year":2005},{"id":70,"shortName":"06X","longName":"Summer 2006","year":2006},{"id":71,"shortName":"06S1","longName":"Semester 1 2006","year":2006},{"id":72,"shortName":"06S2","longName":"Semester 2 2006","year":2006},{"id":73,"shortName":"07X","longName":"Summer 2007","year":2007},{"id":74,"shortName":"07S1","longName":"Semester 1 2007","year":2007},{"id":75,"shortName":"07S2","longName":"Semester 2 2007","year":2007},{"id":76,"shortName":"08X","longName":"Summer 2008","year":2008},{"id":77,"shortName":"08S1","longName":"Semester 1 2008","year":2008},{"id":78,"shortName":"08S2","longName":"Semester 2 2008","year":2008},{"id":79,"shortName":"09X","longName":"Summer 2009","year":2009},{"id":80,"shortName":"09S1","longName":"Semester 1 2009","year":2009},{"id":81,"shortName":"09S2","longName":"Semester 2 2009","year":2009},{"id":82,"shortName":"10X","longName":"Summer 2010","year":2010},{"id":83,"shortName":"10S1","longName":"Semester 1 2010","year":2010},{"id":84,"shortName":"10S2","longName":"Semester 2 2010","year":2010},{"id":85,"shortName":"11X","longName":"Summer 2011","year":2011},{"id":86,"shortName":"11S1","longName":"Semester 1 2011","year":2011},{"id":87,"shortName":"11S2","longName":"Semester 2 2011","year":2011},{"id":88,"shortName":"12X","longName":"Summer 2012","year":2012},{"id":89,"shortName":"12S1","longName":"Semester 1 2012","year":2012},{"id":90,"shortName":"12S2","longName":"Semester 2 2012","year":2012},{"id":91,"shortName":"13X","longName":"Summer 2013","year":2013},{"id":92,"shortName":"13S1","longName":"Semester 1 2013","year":2013},{"id":93,"shortName":"13S2","longName":"Semester 2 2013","year":2013},{"id":94,"shortName":"14X","longName":"Summer 2014","year":2014},{"id":95,"shortName":"14S1","longName":"Semester 1 2014","year":2014},{"id":96,"shortName":"14S2","longName":"Semester 2 2014","year":2014},{"id":97,"shortName":"15X","longName":"Summer 2015","year":2015},{"id":98,"shortName":"15S1","longName":"Semester 1 2015","year":2015},{"id":99,"shortName":"15S2","longName":"Semester 2 2015","year":2015},{"id":100,"shortName":"16X","longName":"Summer 2016","year":2016},{"id":101,"shortName":"16S1","longName":"Semester 1 2016","year":2016},{"id":102,"shortName":"16S2","longName":"Semester 2 2016","year":2016},{"id":103,"shortName":"17X","longName":"Summer 2017","year":2017},{"id":104,"shortName":"17S1","longName":"Semester 1 2017","year":2017},{"id":105,"shortName":"17S2","longName":"Semester 2 2017","year":2017},{"id":106,"shortName":"18X","longName":"Summer 2018","year":2018},{"id":107,"shortName":"18S1","longName":"Semester 1 2018","year":2018},{"id":108,"shortName":"18S2","longName":"Semester 2 2018","year":2018},{"id":109,"shortName":"19X","longName":"Summer 2019","year":2019},{"id":110,"shortName":"19T1","longName":"Term 1 2019","year":2019},{"id":111,"shortName":"19T2","longName":"Term 2 2019","year":2019},{"id":112,"shortName":"19T3","longName":"Term 3 2019","year":2019},{"id":113,"shortName":"20X","longName":"Summer 2020","year":2020},{"id":114,"shortName":"20T1","longName":"Term 1 2020","year":2020},{"id":115,"shortName":"20T2","longName":"Term 2 2020","year":2020},{"id":116,"shortName":"20T3","longName":"Term 3 2020","year":2020},{"id":117,"shortName":"21X","longName":"Summer 2021","year":2021},{"id":118,"shortName":"21T1","longName":"Term 1 2021","year":2021},{"id":119,"shortName":"21T2","longName":"Term 2 2021","year":2021},{"id":120,"shortName":"21T3","longName":"Term 3 2021","year":2021},{"id":121,"shortName":"22X","longName":"Summer 2022","year":2022},{"id":122,"shortName":"22T1","longName":"Term 1 2022","year":2022},{"id":123,"shortName":"22T2","longName":"Term 2 2022","year":2022},{"id":124,"shortName":"22T3","longName":"Term 3 2022","year":2022},{"id":125,"shortName":"23X","longName":"Summer 2023","year":2023},{"id":126,"shortName":"23T1","longName":"Term 1 2023","year":2023},{"id":127,"shortName":"23T2","longName":"Term 2 2023","year":2023},{"id":128,"shortName":"23T3","longName":"Term 3 2023","year":2023},{"id":129,"shortName":"24X","longName":"Summer 2024","year":2024},{"id":130,"shortName":"24T1","longName":"Term 1 2024","year":2024},{"id":131,"shortName":"24T2","longName":"Term 2 2024","year":2024},{"id":132,"shortName":"24T3","longName":"Term 3 2024","year":2024}]
```
